lib/index.js

```diff
@@ -315,6 +316,26 @@ Socket.prototype.maxreconn  = opt('maxreconn');
 Socket.prototype.sndprio    = opt('sndprio');
 Socket.prototype.rcvprio    = opt('rcvprio');
 
+/* chan and rmchan sockopt methods. only relevant for subscription sockets */
+Socket.prototype.chan       = function(){
+  var args = arguments;
```

Comment:
Oh I did that since otherwise writing `arguments` makes this line quite long for readability, are you sure?
```js
if(nn.Setopt(this.binding, nn.NN_SUB, nn.NN_SUB_SUBSCRIBE, args[channels]) == 0)
```